projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -188,7 +192,19 @@ export default function PublicizeForm( {
 							/>
 						</>
 					) }
-					{ isEnhancedPublishingEnabled && <MediaSection /> }
+					{ isEnhancedPublishingEnabled && (
+						<MediaSection
+							disabled={ isSocialImageGeneratorEnabled && isImageGeneratorEnabled }
```

Comment:
Super minor nitpick and we can address it later, `isImageGeneratorEnabled` should perhaps be renamed to `isImageGeneratorSelected`, the word 'enabled' makes me think if the feature is enabled or not.